### ISSUE TITLE
Patrick's test site is not usable for Randy

### ISSUE THREAD

**Issue by patrickzurek** [eXtensibleCatalog/Drupal-Toolkit#93]
JIRA issue created by: rcook
Originally opened: 2012-10-16 02:20 PM

Issue body:

I have tried working with the test site below that Patrick setup to test the harvesting of aggregated records and I can't.

http://xc-zurek.carli.illinois.edu/xc_dev/ 

The site is not usable for me, but it seems to work perfectly well for Patrick.

For me, I do a search, say "boolean" and then get two results, I click to full record and it freezes on me.  I cannot go back to main page or anything.

I cannot see the XC Schema view to test that the right FRBR levels ar being created.   

In short, I can't help move this topic forward if I can't test it.

[~pkiraly] or [~mwesley] or [~pzurek]  can you help please.

**Comment by patrickzurek** [comment]
JIRA Comment by user: rcook
JIRA Timestamp: 2012-10-16 04:12 PM

Comment body:

Seems to be a bug in IE
